packages/@aws-cdk/aws-servicecatalog/lib/constraints.ts

```diff
@@ -0,0 +1,30 @@
+import { AcceptLanguage } from './common';
+
+/**
+ * Properties for governance mechanisms and constraints.
+ */
+export interface CommonConstraintOptions {
+  /**
+   * The language code. Configures the language for error messages
+   * from service catalog.
+   * @default - No accept language provided
```

Comment:
Hmmm. Isn't the default English by any chance?

Reply:
You don't have to provide an `acceptLanguage` to the constructs.  Currently the internal default SC behavior is english, but that could change.  Do you think we should add a default here?  I was following the Cfn resource for parameters optionality.

Reply:
No, don't default here. Just update the documentation that the default is English.